apps/webapp/src/components/staking/account/delegation-value-view/index.tsx

```diff
@@ -15,19 +15,20 @@ import { getAmount, getValidatorInfoFromValueView, joinLoHiAmount } from '@penum
 export const DelegationValueView = ({
   valueView,
   votingPowerAsIntegerPercentage,
-  canDelegate,
```

Comment:
Moving the `canDelegate` and `canUndelegate` logic down into the `<StakingActions />` component, since that component now know our balance of staking and delegation tokens and can thus determine for itself whether we can delegate or undelegate.